src/test/java/de/tum/in/www1/artemis/notification/SingleUserNotificationServiceTest.java

```diff
@@ -315,21 +315,11 @@ void testNotifyUserAboutAssessedExerciseSubmission() {
     }
 
     /**
-     * Test for checkNotificationForAssessmentExerciseSubmission method with an undefined release date
-     */
-    @Test
-    void testCheckNotificationForAssessmentExerciseSubmission_undefinedAssessmentDueDate() {
-        exercise = TextExerciseFactory.generateTextExercise(null, null, null, course);
-        singleUserNotificationService.checkNotificationForAssessmentExerciseSubmission(exercise, user, result);
-        verify(singleUserNotificationService).checkNotificationForAssessmentExerciseSubmission(exercise, user, result);
-    }
-
-    /**
-     * Test for checkNotificationForExerciseRelease method with a current or past release date
+     * Test for checkNotificationForExerciseRelease method with a past assessment due date
      */
     @Test
     void testCheckNotificationForAssessmentExerciseSubmission_currentOrPastAssessmentDueDate() {
```

Comment:
```suggestion
    void testCheckNotificationForAssessmentExerciseSubmission_pastAssessmentDueDate() {
```
Also, why did you change the assessment due date to `ZonedDateTime.now().minusMinutes(1)` ? I'm guessing it was flaky

Reply:
Yes, the production code checks `now.isAfter(assessementDueDate)` and the test was flaky if the same timestamp was now and the saved assessment due date.